docs/testbed/README.testbed.SnappiTests.md

```diff
@@ -0,0 +1,344 @@
+# Running Snappi Tests and Snappi Configuration
+
+### Author
+
+| Version | Name          | Role          |
+| ------- | ------------- | ------------- |
+| 1.0     | Dev Ojha      | Spec Guide    |
+
+
+## 1 Physical Topology
+
+### 1.1 Purpose
+
+Each DUT front-panel port is cabled 1-to-1 to a traffic-generator port of equal speed so that Snappi test scripts can exercise link-level features and inject/measure traffic end-to-end.
+
+### 1.2 Sample link map
+
+| StartDevice | StartPort | EndDevice | EndPort | Bandwidth (Mb/s) | VLAN ID | VLAN Mode |
+| ----------- | --------- | --------- | ------- | ---------------- | ------- | --------- |
+| `dut1`      | Ethernet1 | `tgen1`   | P1      | 100 000          |         | Access    |
+| `dut1`      | Ethernet2 | `tgen1`   | P2      | 100 000          |         | Access    |
+
+The CSV file that carries this information is called **`sonic_tgen_links.csv`** and must reside next to the Ansible playbooks (`/ansible` directory) so that `conn_graph_facts.py` can feed the tests.
+
+---
+
+## 2 Testbed Configuration Files
+
+### 2.1 `testbed.yaml`
+
+Each block describes one logical topology; the `ptf_ip` is the management address of the traffic-generator’s API-Docker container. If you add multiple testbeds/devices, you can use the same Docker API Server (more below), so the `ptf_ip` would be the same for these testbeds.
+
+```yaml
+- conf-name: lab-t0-dut1
+  group-name: lab-grp
+  topo: t0
+  ptf_image_name: docker-ptf
+  ptf: tgen1
```

Comment:
Is this tgen1 ixia IP or having reference to other csv or yaml files for the IP?